### PR TITLE
Ubuntu clarification

### PR DESCRIPTION
**Pull request by jburel**
Clarify installation
cc @rgozim

### CODE REVIEW

File: omero/sysadmins/unix/server-ubuntu-ice36.rst [hunk 1]
```diff
@@ -66,7 +66,9 @@ Create an omero system user, and a directory for the OMERO repository:
     :start-after: #start-step02
     :end-before: #end-step02
 
-Create a database user and initialize a new database for OMERO:
+Create a database user and initialize a new database for OMERO. For the commands
+below to work, add if required the user you are currently logged to the newly
```
Comment (mtbc) [inline omero/sysadmins/unix/server-ubuntu-ice36.rst]: "Currently logged" is missing some words, maybe "currently logged in as" or simply "the current system user" or something? There may be a better option I'm not yet seeing.

**Comment by jburel** [comment]
@mtbc better?

**Comment by mtbc** [comment]
Yes, thank you.

**Comment by jburel** [comment]
merging in that case

**Pull request merged**